operator/gitops/argocd/pipeline-service/tekton-results/namespace.yaml

```diff
@@ -6,4 +6,4 @@ metadata:
   labels:
     argocd.argoproj.io/managed-by: openshift-gitops
   annotations:
-    argocd.argoproj.io/sync-wave: "0"
+    argocd.argoproj.io/sync-wave: "-1"
```

Comment:
should we add a comment here too since a -1 sync-wave is "atypical" IIUC @enarha @Roming22 ?

Reply:
Done.